libs/wingsdk/API.md

```diff
@@ -890,6 +890,181 @@ with a fresh copy without any consequences.
 ---
 
 
+### Schedule <a name="Schedule" id="@winglang/sdk.cloud.Schedule"></a>
```

Comment:
Need to also update /docs/04-reference/wingsdk-api.md.
You don't need to do these changes manually, of course. Just run `npm run build` from the libs/wingsdk folder and all your code changes will be reflected in the generated docs API.md and copied to wingsdk-api.md

Reply:
done